hack/build-antrea-windows-all.sh

```diff
@@ -70,6 +70,7 @@ NANOSERVER_VERSION=$(head -n 1 build/images/deps/nanoserver-version)
 CNI_BINARIES_VERSION=$(head -n 1 build/images/deps/cni-binaries-version)
 GO_VERSION=$(head -n 1 build/images/deps/go-version)
 WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10)
+OVS_VERSION=2.15.2 #TBD For windows ovs version build
```

Comment:
To avoid using the hardcoded value in dockerfile, you can set this variable in build/images/deps.

Reply:
The file vi build/images/deps/ovs-version has already a version inside it ( 2.17.5) but this ovs version is not built for windows

Reply:
This version is for linux ovs, we can name a new value for windows ovs because their versions can be different.